src/components/comments/CommentList.js

```diff
@@ -1,28 +1,38 @@
-import React, { useEffect, useContext } from "react"
+import React, { useEffect, useContext, useState } from "react"
 import ListGroup from "react-bootstrap/ListGroup"
 import { CommentContext } from "./CommentProvider"
 import { Comment } from "./Comment"
 
 export const CommentList = props => {
-  const { postId } = props
-
   const { comments, getCommentsByPostId } = useContext(CommentContext)
-
+  console.log('Here', comments)
+  
   useEffect(() => {
+    const postId = parseInt(props.match.params.postId);
     getCommentsByPostId(postId)
   }, [])
 
   return (
-    <div className="commentList" style={{ maxHeight: '250px', overflowY: 'scroll' }}>
-      <ListGroup>
-        {
-          comments.map(c => (
-            <ListGroup.Item key={c.id}>
-              <Comment comment={c} />
-            </ListGroup.Item>
-          ))
-        }
-      </ListGroup>
+    <div>
+      <div className="commentList" style={{ maxHeight: '250px', overflowY: 'scroll' }}>
+        <ListGroup>
+
+          {
+            comments.map(c => (
+              <ListGroup.Item key={c.id}>
+                <Comment comment={c.content} />
```

Comment:
It seems like this component isn't really doing anything anymore... most of the stuff it was rendering has been moved up to the `CommentList`. It seems like we could just remove this component from being rendered here, and also probably even delete the `Component.js` component altogether if we are just going to render all this stuff here.